tools/KoreBuildSettings.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Was korebuild.json not being read before?

Reply:
It was read by the bootstrapper only. This is the first use case for reading korebuild.json from KoreBuild. I expect there will be more.